src/promptflow/promptflow/_utils/capability_util.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+
+from promptflow._utils.utils import load_json
+
+CAPABILITY_LIST_FILE = Path(__file__).parent / "capability_list.json"
+
+
+def get_capability_list():
+    capability_list = load_json(CAPABILITY_LIST_FILE)
+    return capability_list
```

Comment:
Do we want to add a dataclass for capalitity? So that we can check typos in that class. (i.e. Someone may accidentally type "Reday" for "Ready", etc)

Reply:
+1